releases/release-1.33/release-notes/maps/pr-128929-map.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 128929
+releasenote:
+  text: "Fixed a bug where the kube-proxy `EndpointSliceCache` memory
+    was leaked."
```

Comment:
Does this need the double quotes?

Reply:
Not mandatory. removed the double quotes.